bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx

```diff
@@ -150,12 +150,16 @@ class VideoList extends Component {
     return users.map((user) => {
       const isFocused = focusedId === user.id;
       const isFocusedIntlKey = !isFocused ? 'focus' : 'unfocus';
-      const actions = [{
-        label: intl.formatMessage(intlMessages[`${isFocusedIntlKey}Label`]),
-        description: intl.formatMessage(intlMessages[`${isFocusedIntlKey}Desc`]),
-        onClick: () => this.handleVideoFocus(user.id),
-        disabled: users.length < 2,
-      }];
+      let actions = [];
+
+      if (users.length > 2) {
+        actions = [{
+          label: intl.formatMessage(intlMessages[`${isFocusedIntlKey}Label`]),
+          description: intl.formatMessage(intlMessages[`${isFocusedIntlKey}Desc`]),
+          onClick: () => this.handleVideoFocus(user.id),
+          disabled: users.length < 3,
```

Comment:
We have ` if (users.length > 2) {`, why do we need to make the `disabled: users.length < 3,` ?
Looks like it should always be enabled